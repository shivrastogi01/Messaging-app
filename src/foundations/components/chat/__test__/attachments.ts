import exampleGiphy from "./exampleGiphy";

export const link = {
  url: "https://pubnub.com",
  provider: {
    name: "PubNub",
    url: "https://pubnub.com"
  },
  title:
    "Realtime Communication APIs for Chat, Notifications, Geolocation, and Collaboration | PubNub",
  description:
    "Leverage the power of hosted realtime APIs to build and augment connected experiences including in-app chat, mapping, device control and more on a massive scale.",
  image:
    "https://images.ctfassets.net/3prze68gbwl1/3roUbsKJAysxW6gUUxczrY/89eeaa8940ccf2ed10b348adaca99b64/pubnub-powered-apps.png",
  icon: "https://www.pubnub.com/images/favicon.png"
} as const;

export const image = {
  source:
    "https://images.ctfassets.net/3prze68gbwl1/7Mtx0mNP5byeuqrTr8gBCS/0d4ca318dc54edc7a17cfa9bf9987ecd/pubnub-chat-sdk-messaging-api.png"
};

export const gif = {
  title: "Giphy Logo",
  gif: exampleGiphy
};
