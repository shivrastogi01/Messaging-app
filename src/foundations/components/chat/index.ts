export * from "./text-message/text-message";
export * from "./image-message/image-message";
export * from "./link-message/link-message";
export * from "./giphy-message/giphy-message";
export * from "./avatar/avatar";
