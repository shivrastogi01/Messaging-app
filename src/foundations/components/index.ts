export * from "./presentation";
export * from "./layout";
