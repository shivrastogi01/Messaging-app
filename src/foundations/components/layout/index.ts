export * from "./styled-box/styled-box";
export * from "./scroll-view/scroll-view";
export * from "./flex-row/flex-row";
export * from "./flex-column/flex-column";
export * from "./modal/modal";
export * from "./drawer/drawer";
export * from "./list-item/list-item";
export * from "./dropdown/dropdown";
