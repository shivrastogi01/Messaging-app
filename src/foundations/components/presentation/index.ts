export * from "./heading/heading";
export * from "./label/label";
export * from "./title/title";
export * from "./icon/icon";
export * from "./button/button";
export * from "./input/input";
export * from "./textarea/textarea";
