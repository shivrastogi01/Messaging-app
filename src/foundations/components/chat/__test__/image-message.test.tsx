import React from "react";
import { ImageMessage } from "foundations/components/chat";
import renderWithTheme from "main/testing/render-with-theme";
import { image } from "./attachments";

describe("Images Message", () => {
  test("example matches the snapshot", () => {
    const { container } = renderWithTheme(<ImageMessage image={image} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
