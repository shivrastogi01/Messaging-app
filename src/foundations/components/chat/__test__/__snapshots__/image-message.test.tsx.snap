// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Images Message example matches the snapshot 1`] = `
.c0 {
  border-radius: 10px;
  width: 100%;
  height: 100%;
  object-fit: cover;
  object-position: center center;
}

<img
  alt="media from [object Object]"
  class="c0"
  src="[object Object]"
/>
`;
